packages/client/hmi-client/src/temp/Clipboard.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+	<main style="width: 90%; height: 90%; overflow: scroll">
+		<h4>clipboard test</h4>
+
+		<tera-stratified-matrix
+			v-if="ready"
+			:mmt="mmt"
+			:mmt-params="mmtParams"
+			:id="'beta'"
+			:stratified-matrix-type="StratifiedMatrix.Parameters"
+			:should-eval="false"
+			:matrix-type="'subjectControllers'"
+		/>
+	</main>
+</template>
+
+<script setup lang="ts">
+import { onMounted, onUnmounted, ref } from 'vue';
+import { getMMT } from '@/services/model';
+import TeraStratifiedMatrix from '@/components/model/petrinet/model-configurations/tera-stratified-matrix.vue';
+import { emptyMiraModel } from '@/model-representation/mira/mira';
+import type { Model } from '@/types/Types';
+import type {
+	MiraModel,
+	MiraTemplateParams,
+	TemplateSummary
+} from '@/model-representation/mira/mira-common';
+import { StratifiedMatrix } from '@/types/Model';
+import { updateParameter } from '@/model-representation/service';
+
+import * as sirJSON from '@/examples/strata-example.json';
+
+const model = ref<Model>();
+const mmt = ref<MiraModel>(emptyMiraModel());
+const mmtParams = ref<MiraTemplateParams>({});
+const ready = ref(false);
+
+/**
+ * Given a matrix-like CSV (with row and column labels), update the model with specified values.
+ *
+ * For example, with:
+ * ,A,B
+ * C,1,2
+ * D,3,4
+ *
+ * Will yield (A, C, 1), (A, D, 3), (B, C, 2) and (B, D, 4) update tuples
+ *
+ **/
+const updateByMatrixCSV = async (
+	model: Model,
+	paramLocationMap: Map<string, TemplateSummary[]>,
+	matrixType: string,
+	text: string
+) => {
+	// Parse
+	const rows = text.split('\n');
+	const columns = rows[0].split(',');
```

Comment:
If the first row is not the column's names, what would happen? Should we test for that or it's too much?

Reply:
it will break, we will need to craft different parsers for different types of CSV-formats